pkg/rhcos/vmware.go

```diff
@@ -0,0 +1,44 @@
+package rhcos
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/pkg/errors"
+
+	"github.com/openshift/installer/pkg/types"
+)
+
+// VMware fetches the URL of the Red Hat Enterprise Linux CoreOS release.
+func VMware(ctx context.Context, arch types.Architecture) (string, error) {
+	meta, err := fetchRHCOSBuild(ctx, arch)
+	if err != nil {
+		return "", errors.Wrap(err, "failed to fetch RHCOS metadata")
+	}
+
+	base, err := url.Parse(meta.BaseURI)
+	if err != nil {
+		return "", err
+	}
+
+	image, err := url.Parse(meta.Images.VMware.Path)
+	if err != nil {
+		return "", err
+	}
+
+	baseURL := base.ResolveReference(image).String()
+
+	// TODO: Get Uncompressed SHA256 into rhcos.json
+	// Attach sha256 checksum to the URL.  Always provide the
+	// uncompressed SHA256; the cache will take care of
+	// uncompressing before checksumming.
+	//baseURL += "?sha256=" + meta.Images.VMware.UncompressedSHA256
```

Comment:
since OVA file is not uncompressed, the SHA (https://github.com/openshift/installer/pull/3299/files#diff-89719a4e5e27791c48d41ee4b359b7daR46) should just work. right?